test/integration/test-video-players-helper.js

```diff
@@ -50,8 +50,6 @@ export function runVideoPlayerIntegrationTests(
    */
   const FRAME_HEIGHT = 1000;
 
-  const TIMEOUT = 20000;
```

Comment:
These tests are supposed to run for multiple element types, not only `<amp-video>`. Others, for example `<amp-youtube>`, load external URLs into an iframe so we get into flakier territory, hence such a large timeout.

See [test-video-players.js](https://github.com/ampproject/amphtml/blob/b5c85d7510aadcbc4562228e795c7d94d7ddc562/test/integration/test-video-players.js) for all definitions. All of these but `<amp-video>` are skipped, so we need a larger TODO to replace their strategy.

In the meantime I'm totally on board of making the single running test faster, but I suggest that we make the timeout optional rather than get rid of it entirely. This would be well suited as an argument for `runVideoPlayerIntegrationTests`. We can default it to `1000`, and change users except `amp-video` to pass the original value of `20000`.

Reply:
Thanks for the detailed explanation. I'd like to push back on one point if I may: including 20 second long integration tests in every CI build is very expensive, so I don't think we should go back in that direction. A couple alternatives that come to mind:

1. Enhance our local test server to serve the contents and return a short video of each type (and add an entry to the `hosts` file)
2. Convert these long running tests to manual QA verifications

Based on this, is it fair to ask that we remove the 20 second long timeout? (I think a long term goal of ours should be to completely eliminate long running tests from CI.)

Reply:
> so that we can run the tests locally.

> while still allowing them to run locally because one can manually unskip/change the timeout.

Both these sentences have me convinced. Agree with leaving in the ability to run these tests locally.

Reply:
Being able to run the tests locally after unskipping them makes sense to me. I'd like to leave the timeout as `20000ms` for just the skipped tests though so it is clear that it needs changing.